### DISCUSSION TITLE
iFrame Issue on WordPress Site: Ticket Category Selection Not Working and Strict-Origin When Cross-Origin Error

### DISCUSSION THREAD

**Discussion by bajzikgyuri** [alfio-event/alf.io#1211] [Q&A]
Hello everyone,

I have successfully embedded my Alf.io instance using an iFrame on my WordPress site, and it seems to load up without any issues. However, I've encountered a problem when trying to select a ticket category and proceed to the next step. Nothing happens when I click "next," and upon checking my browser log, I see the following error messages:

[URL]/api/v2/public/event/[event-name]/check-discount 404 (Not Found)
Strict-Origin when Cross-Origin error
I also noticed a "404 Not Found" error when sending a POST request to the following URL:
[URL]/api/v2/public/event/[event-name]/reserve-tickets?lang=en

I have redacted the actual URL and event name for privacy reasons. It's important to mention that I am using a different domain for my Alf.io instance than my WordPress site.

Has anyone else experienced these issues when embedding an Alf.io instance in a WordPress site using an iFrame? If so, could you please share any suggestions or solutions to resolve these problems?

I appreciate any help you can provide. Thank you in advance!

Best regards,
George

**Comment by cbellone** [comment]
Hi George,

I assume you have already configured the relevant "embedding options" on your alf.io instance

![image](https://user-images.githubusercontent.com/3385346/235848560-4059ea5f-63d3-43ee-baa2-55037d6ebcbb.png)

The current version (2.0-M4-xx) allows you to embed alf.io only if you're using the same root domain for both container and embedded resources (i.e. embed tickets.mydomain.com within website.mydomain.com).
M5 will fix this
